Jenkinsfile

```diff
@@ -0,0 +1,19 @@
+pipeline {
+  agent any
+ 
+  tools {nodejs "node"}
+ 
+  stages {
+    stage('Install dependencies') {
+      steps {
+        sh 'npm install'
+      }
+    }
+     
+    stage('Test') {
+      steps {
+         sh 'npm test'
```

Comment:
Do we also want to run linting as part of the jenkins job?

Reply:
sure!  whats the command?

Reply:
`npm lint`

Reply:
added